canonical/greenhouse-automations
# Added EEOC questions to job posts that will be posted to USA

## Done
- Separated USA cities 
- Added EEOC questions if the job post will be posted to a city in the USA

## QA
- Checkout the feature branch
- Install dependencies with `yarn`
- Run ` ts-node ./src/index.ts replicate -i`
- Select "americas" for region
- Select necessary fields
- Verify EEOC checkboxes in job posts that are posted to the USA are checked.
- Verify EEOC checkboxes are not selected for other countries.

Fixes #65
